backend/calendar/calendar-api.js

```diff
@@ -220,33 +131,6 @@ module.exports = (app) => {
 
       if (conflictingEvents && conflictingEvents.length > 0) {
         onInvalidUserInput(res, 'Start date and end date overlap with another event');
```

Comment:
Thoughts on if there are two events happening at the same time? Maybe this check should warn the admin rather than prevent them from doing it?

Reply:
This was up in the air for me too. I feel like we should just allow it to make the process easier for admins just in case they need to do something special. Thoughts?

Reply:
Yeah I agree with that!